tool/esm_resolver_generator.js

```diff
@@ -0,0 +1,65 @@
+var fs = require("fs");
+
+function buildResolver() {
```

Comment:
can you import these functions from Makefile.dryice.js?

Reply:
I changed code to import, hope that would work